dashboard/config/locales/scripts.en.yml

```diff
@@ -29586,9 +29593,40 @@ en:
               name: Project   Make a Library Part 3
               description_student: The third day of a three day project where students build and test their own libraries.
               description_teacher: Students complete their library project, finalizing their code and writing written responses explaining the way one of the functions in their library works.
+            Assessment Day:
```

Comment:
I think cloning may have failed on this lesson, in particular on cloning the sublevels of the assessment on [this line](https://github.com/code-dot-org/code-dot-org/blob/cda29eef40c5739f71620591b117528a242fa527/dashboard/app/models/levels/level.rb#L667). The level group itself got cloned but it looks like the individual levels didn't -- see https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/scripts/csp_unit_7_ap_style_assessment_2022.level_group. Because this level already exists locally, it likely didn't try to re-clone these levels, so the issue didn't reappear (and at this point, won't on levelbuilder either). We can keep doing this locally, but I think we need to fix this

Reply:
I can get a much smaller repro by running, in dashboard-console:
```
level2022 = Level.find_by_name('CSP Unit 7 AP-Style Assessment 2022')
level2021 = Level.find_by_name('CSP Unit 7 AP-Style Assessment _2021')
level2022.destroy!
level2021.clone_with_suffix('2022')
```

I think if you can get that to run without errors, we may be able to get this script into a good state. In particular, we should be able to run, on levelbuilder:
```
Level.find_by_name('CSP Unit 7 AP-Style Assessment 2022').destroy!
script.find_by_name('csp7-2021').clone_migrated_unit(...)
```
and feel relatively confident in it. (We could also delete `CSP Unit 7 AP-Style Assessment 2022` from the [level edit page](https://levelbuilder-studio.code.org/levels/32434))

Reply:
Oh thanks, Bethany. I'll try that––will keep you updated